doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/getting-started.ipynb

```diff
@@ -432,74 +371,10 @@
-    "\n",
-    "Assuming the port-forwarding process described above is still running, you may view the {ref}`ray-dashboard` by visiting `localhost:8265` in you browser.\n",
-    "\n",
-    "The dashboard port will not be used in the rest of this guide. You may stop the port-forwarding process if you wish.\n",
-    "\n",
-    "(kuberay-client)=\n",
-    "### Accessing the cluster using Ray Client\n",
-    "\n",
-    "[Ray Client](https://docs.ray.io/en/latest/cluster/ray-client.html) allows you to interact programatically with a remote Ray cluster using the core Ray APIs.\n",
-    "To try out Ray Client, first make sure your local Ray version and Python minor version match the versions used in your Ray cluster. The Ray cluster in our example is running Ray 2.0.0 and Python 3.7, so that's what we'll need locally. If you have a different local Python version and would like to avoid changing it, you can modify the images specified in the yaml file `ray-cluster.autoscaler.yaml`. For example, use `rayproject/ray:2.0.0-py38` for Python 3.8.\n",
-    "\n",
-    "After confirming the Ray and Python versions match up, the next step is to port-forward the Ray Client server port (10001 by default).\n",
-    "If you are following along in a Jupyter notebook, execute the following command in a separate shell."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "d1f4154f",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Execute this in a separate shell.\n",
-    "! kubectl port-forward service/raycluster-autoscaler-head-svc 10001:10001"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "6ab3bdfe",
-   "metadata": {},
-   "source": [
-    "Now that we have port-forwarding set up, we can connect to the Ray Client from a local Python shell as follows:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "c30245c2",
+   "id": "f8453b2a",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "import ray\n",
-    "import platform\n",
-    "\n",
-    "ray.init(\"ray://localhost:10001\")\n",
-    "\n",
-    "# The network name of the local machine.\n",
-    "local_host_name = platform.node()\n",
-    "\n",
-    "# This is a Ray task.\n",
-    "# The task will returns the name of the Ray pod that executes it.\n",
-    "@ray.remote\n",
-    "def get_host_name():\n",
-    "    return platform.node()\n",
-    "\n",
-    "# The task will be scheduled on the head node.\n",
-    "# Thus, this variable will hold the head pod's name.\n",
-    "remote_host_name = ray.get(get_host_name.remote())\n",
-    "\n",
-    "print(\"The local host name is {}\".format(local_host_name))\n",
-    "print(\"The Ray head pod's name is {}\".format(remote_host_name))\n",
-    "\n",
-    "# Disconnect from Ray.\n",
-    "ray.shutdown()"
+    "For a more detailed guide on using Ray Jobs to run applications on a Ray Cluster, check out the [quickstart guide](jobs-quickstart-under-construction)"
```

Comment:
Optional, idea: Since the Ray cluster is already set up for it, it could make sense to briefly mention autoscaling and link the autoscaling quickstart.

Reply:
I'll do autoscaling changes in a follow-up PR.